### ISSUE TITLE
ref on ScrollSyncPane child is not working

### ISSUE THREAD

**Issue by kot-shrodingera** [okonet/react-scroll-sync#79]
How can I use `ref` on `ScrollSyncPane` child? It is always `undefined`. Maybe that's because `ScrollSyncPane` is cloning child in `render` function, passing `childRef` property, which is never set?

**Comment by okonet** [comment]
Yeah that could be the reason. I found out over time the cloning children should be avoided but I'm currently not really maintaining this lib. So if you want to submit a PR feel free to do so.

**Comment by vanntrong** [comment]
> 

Hi okonet, I have created a pr for this issue, please check it https://github.com/okonet/react-scroll-sync/pull/90

**Comment by themcmxciv** [comment]
having same issue, in my case adding **React Fragment** solved the issue.

![Screen Shot 2023-06-05 at 14 03 14](https://github.com/okonet/react-scroll-sync/assets/26102923/60f80857-ab51-4800-95b6-6a6d8cd46f80)
